_pkgdown.yml

```
@@ -178,11 +178,21 @@ reference:
 
 
   redirects:
-    - ["articles/cards.html#fixed-sizing", "articles/cards.html#scrolling-contents"]
-    - ["articles/cards.html#responsive-sizing", "articles/cards.html#filling-contents"]
-    - ["articles/cards.html#fixed-responsive-sizing", "articles/cards.html#scrolling-filling"]
-    - ["articles/cards.html#dynamic-rendering-shiny", "articles/cards.html#shiny"]
     - ["reference/nav.html", "reference/nav-items.html"]
     - ["reference/navs.html", "reference/navset.html"]
-    - ["articles/custom-theming.html", "articles/theming.html"]
-    - ["articles/bslib.html", "articles/any-project.html"]
+    - ["articles/custom-theming.html", "articles/theming/index.html"]
+    - ["articles/bslib.html", "articles/any-project/index/index.html"]
```

Seems like the other links have the same problem?

```suggestion
    - ["articles/bslib.html", "articles/any-project/index.html"]
```